atom/app/node_main.cc

```diff
@@ -57,10 +57,12 @@ int NodeMain(int argc, char *argv[]) {
     if (node_debugger.IsRunning())
       env->AssignToContext(v8::Debug::GetDebugContext());
 
-#if defined(OS_WIN)
     mate::Dictionary process(gin_env.isolate(), env->process_object());
+#if defined(OS_WIN)
     process.SetMethod("log", &AtomBindings::Log);
 #endif
+    process.SetMethod("startCrashReporter", &AtomBindings::StartCrashReporter);
```

Comment:
Thinking a little down the line, this might be better available as `process.crashReporter.start()`. That way we can easily add additional `crashReporter` APIs to this object in the future if needed.